src/controller/data_model/controller-clusters.zap

```diff
@@ -5593,6 +5593,66 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "abs min heat setpoint limit",
+              "code": 3,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "0x02BC",
+              "reportable": 0,
+              "minInterval": 0,
```

Comment:
neeeds ranges from the spec

Reply:
Marking resolved - even though some ranges seem reasonable, I cannot see them in the new spec - spec html file I was reading seems to have been generated previously.

It does seem unfortunate we have min/max that are bogus (not even 0x7FFF or 0xFFFFF) in the zap files.

Reply:
Same comment as above - Issue created.